README.md

```diff
@@ -47,6 +47,14 @@ and it will require the sns:Publish action. Here is an [example](https://github.
 
 A test [google group](https://groups.google.com/a/guardian.co.uk/g/anghammarad.test.alerts) has been created, so that you can test your notification. The stack for this is called "testing-alerts". You can also find this in the Anghammarad config in S3.
 
+You can try out the `CODE` deployment before promoting to `PROD`, by obtaining "Deploy Tools" developer credentials from Janus and running:
```

Comment:
IIUC the `dev` project (which I learnt about thanks to this PR!) will exercise most of the Anghammarad functionality locally but will only interact with `CODE` AWS resources in order to pull the config file from S3.

In other words, it will not actually invoke the `CODE` AWS Lambda, so there is a small gap here in coverage (i.e. we won't test the 'real' execution environment or IAM permissions). For example I don't think it would have caught the problem we discussed in https://github.com/guardian/anghammarad/pull/78.

I think this is still fine for testing most things, but it's not quite what I'd expect from the description here, so I would probably phrase this as: "You can send a real notification locally by obtaining..."

We could also add: "If you need to invoke the `CODE` Lambda function (for example to test IAM permissions), you can publish a message to Anghammarad's `CODE` SNS topic. The JSON format required for this message can be found [here](https://github.com/guardian/anghammarad/tree/main/anghammarad/src/test/resources).

Reply:
Yeah, you're right I see it calling `Anghammarad.run`. Didn't quite pull the thread far enough yesterday, will PR another update with your suggested changes.